### PR TITLE
Add jest-watch-typeahead to testing setup

### PR DESCRIPTION
**Pull request by tigranpetrossian**
The PR adds [jest-watch-typeahead](https://github.com/jest-community/jest-watch-typeahead) making searching tests easier both via test and file names:

typeahead-preview

## React Boilerplate

Thank you for contributing! Please take a moment to review our [**contributing guidelines**](https://github.com/react-boilerplate/react-boilerplate/blob/master/CONTRIBUTING.md)
to make the process easy and effective for everyone involved.

**Please open an issue** before embarking on any significant pull request, especially those that
add a new library or change existing tests, otherwise you risk spending a lot of time working
on something that might not end up being merged into the project.

Before opening a pull request, please ensure:

- [x] You have followed our [**contributing guidelines**](https://github.com/react-boilerplate/react-boilerplate/blob/master/CONTRIBUTING.md)
- [x] Double-check your branch is based on `dev` and targets `dev` 
- [ ] Pull request has tests (we are going for 100% coverage!)
- [ ] Code is well-commented, linted and follows project conventions
- [ ] Documentation is updated (if necessary)
- [ ] Internal code generators and templates are updated (if necessary)
- [x] Description explains the issue/use-case resolved and auto-closes related issues

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

**IMPORTANT**: By submitting a patch, you agree to allow the project
owners to license your work under the terms of the [MIT License](https://github.com/react-boilerplate/react-boilerplate/blob/master/LICENSE.md).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -130,6 +130,7 @@
     "jest-cli": "24.8.0",
     "jest-dom": "3.1.4",
     "jest-styled-components": "6.3.1",
+    "jest-watch-typeahead": "^0.3.1",
```
Comment (julienben) [inline package.json]: Please pin the version number by removing the ^ (and rerun `npm i` for a small package-lock update).

Comment (tigranpetrossian) [reply package.json]: Whoops, done. `package-lock.json` stayed the same.

**Comment by julienben** [comment]
Thanks for your contribution @tigranpetrossian!

**Comment by tigranpetrossian** [comment]
> Thanks for your contribution @tigranpetrossian!

Thanks for accepting it. More test-related improvements incoming some time soon!

**Pull request merged**